packages/hooks/useMaxResolution.ts

```diff
@@ -22,12 +20,13 @@ export const useMaxResolution = ({
   isLarge = false,
 } = {}) => {
   const { width: windowWidth, height: windowHeight } = useWindowDimensions();
-  const { isSidebarExpanded } = useSidebar();
   const isMobile = useIsMobile();
+
+  // If we have a different width when sidebar is expanded and when it's not, the sidebar will be laggy on certain screens (like FeedScreen)
+  // So this calcul find the bigger width to have the same width no matter the sidebar state
   const contentWidth = useMemo(
```

Comment:
IMHO, useMemo does not gain much in perf. But we can let it, there is no problem.

Reply:
I didn't change the `useMemo` i just explained that if the width change between sidebars states, it will be laggy 👍